pkgs/edge-worker/src/EdgeWorker.ts

```
@@ -29,12 +38,70 @@ export type EdgeWorkerConfig = QueueWorkerConfig;
+   * @param handlerOrFlow - Either a message handler function or a Flow instance
+   * @param config - Configuration options for the worker
+   */
+  static async start<
+    TPayload extends Json = Json,
+    TFlow extends AnyFlow = AnyFlow
+  >(
+    handlerOrFlow: MessageHandlerFn<TPayload> | TFlow,
+    config: EdgeWorkerConfig = {}
+  ): Promise<PlatformAdapter> {
+    if (typeof handlerOrFlow === 'function') {
+      return this.startQueueWorker(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)